tools/show_host_file_entries_for_vnc.rb

```diff
@@ -15,8 +15,9 @@
   if host.ipaddress.ipaddress?
     ipaddress = host.ipaddress
   else
+    require 'socket'
     begin
-      ipaddress = MiqSockUtil.resolve_hostname(host.ipaddress)
+      ipaddress = TCPSocket.gethostbyname(hostname.split(',').first).last
```

Comment:
where did `host.ipaddress` parameter go?

Reply:
Oh good eyes 👀  ...I missed that.

Reply:
Ha, well, why were we passing an IP to a `gethostbyname` call in the first place? Let me refactor this.

Reply:
@chessbyte, @Fryguy Updated.

Reply:
while the variable name is called ipaddress, you'll notice this is inside of the `else` check on `if host.ipaddress.ipaddress?`.  In other words...this is for when there _isn't_ an actual ip address in that value.